### PR TITLE
Update Rule “azure-naming-resources/rule”

### PR DESCRIPTION
**Pull request by wicksipedia**
Automatically generated by Netlify CMS

### CODE REVIEW

File: rules/azure-naming-resources/rule.md [hunk 1]
```diff
@@ -36,8 +36,8 @@ Organizing your cloud assets starts with good names. It is best to be consistent
 
 * All lower case 
 * Use [kebab case (“-“ as a separator)](https://www.ssw.com.au/rules/use-dashes-in-urls)
+* Include an [resource type abbreviation](https://docs.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-best-practices/resource-abbreviations) (so it's easy to find the resource in a script)
```
Comment (lukecookssw) [inline rules/azure-naming-resources/rule.md:39]: Typo "an".
Change to "Include **a** resource type abbreviation"

**Review by lukecookssw — CHANGES_REQUESTED** [review]

File: rules/azure-naming-resources/rule.md (hunk 1 above)
Comment (wicksipedia) [inline rules/azure-naming-resources/rule.md:39]:
```suggestion
* Include a [resource type abbreviation](https://docs.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-best-practices/resource-abbreviations) (so it's easy to find the resource in a script)
```

Comment (wicksipedia) [reply rules/azure-naming-resources/rule.md]: done, next time can you just use the suggested change button

**Review by lukecookssw — APPROVED** [review]
👍

**Pull request merged**